### PR TITLE
Add placeholders for input fields in sign-up component

### PR DESCRIPTION
**Pull request by rochagbriel**

### CODE REVIEW

**Review by ttobollik — CHANGES_REQUESTED** [review]
Well done! A little bit of minor feedback only :)

File: src/app/sign-up/sign-up.component.html [hunk 1]
```diff
@@ -39,6 +40,7 @@ <h4 class="modal-title">Sign Up for a Trip</h4>
         id="lastName"
         formControlName="lastName"
         required
+        placeholder="Enter your last name"
```
[the remark this answers is not in the document]
Comment (rochagbriel) [reply src/app/sign-up/sign-up.component.html]: done!

File: src/app/sign-up/sign-up.component.html [hunk 2]
```diff
@@ -19,6 +19,7 @@ <h4 class="modal-title">Sign Up for a Trip</h4>
         id="firstName"
         formControlName="firstName"
         required
+        placeholder="Enter your first name"
```
[the remark this answers is not in the document]
Comment (rochagbriel) [reply src/app/sign-up/sign-up.component.html]: I couldn't style the option. I tried a few solutions, but they didn't work.

**Review by ttobollik — APPROVED** [review]
Well done. Thanks!

**Pull request merged**